src/backend/InvenTree/part/serializers.py

```diff
@@ -298,7 +297,7 @@ class Meta:
 
     def validate_image(self, value):
         """Check that file is an image."""
-        validate = imghdr.what(value)
+        validate = InvenTree.helpers.TestIfImage(value)
         if not validate:
             raise serializers.ValidationError('File is not an image')
```

Comment:
While you are here, can you please add a translation wrapper around the error message - as it is a user facing string.

```python
raise serializers.ValidationError(_('File is not an image'))
```

Reply:
Done